deluge/ui/console/modes/connectionmanager.py

```diff
@@ -174,28 +170,21 @@ def on_resize(self, rows, cols):
     def read_input(self):
         c = self.stdscr.getch()
 
-        if c > 31 and c < 256:
-            if chr(c) == 'q' and self.inlist:
-                return
+        if is_printable_chr(c):
```

Comment:
Considering it only tests for four different values inside the if, this isn't an issue. Too early in the morning :-)